app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
+    if len(split_command) == 3:
+        if (split_command[0] == "cp"
+                and split_command[1] != split_command[2]):
+            with (open(split_command[1], "r") as file_in,
+                  open(split_command[2], "w") as file_out):
+                content = file_in.read()
+                file_out.write(content)
```

Comment:
There is no need for the `content` variable, pass `file_in.read()` in the `file_out.write()` directly.